.CI/chatterino-installer.iss

```diff
@@ -77,7 +77,7 @@ Source: "{#WORKING_DIR}vc_redist.x64.exe"; DestDir: "{tmp}"; Tasks: vcredist;
 ; NOTE: Don't use "Flags: ignoreversion" on any shared system files
 
 [Icons]
-Name: "{autoprograms}\{#MyAppName}"; Filename: "{app}\{#MyAppExeName}"
+Name: "{autoprograms}\{#MyAppName}"; Filename: "{app}\{#MyAppExeName}"; AppUserModelID: ".Chatterino 2"
```

Comment:
Is `AppUserModelID` the string that appears on top of the notification?

Reply:
No, it is used to associate the shortcut with the app.
I probably add it to the Desktop Shortcut as well.